src/bin/neptune-cli.rs

```diff
@@ -405,6 +405,7 @@ struct Config {
 
 #[tokio::main]
 async fn main() -> Result<()> {
+    display_banner();
```

Comment:
While I love the banner, I think it is noisy in the CLI tool. The CLI should be lean, producing as little output as possible, subject to meeting the user's demands. Please drop this line.